iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/env/RegionMaintainHandler.java

```diff
@@ -693,11 +703,22 @@ public void transferRegionLeader(TConsensusGroupId regionId, TDataNodeLocation o
           (CONF.getSchemaRegionRatisRpcLeaderElectionTimeoutMaxMs()
                   + CONF.getSchemaRegionRatisRpcLeaderElectionTimeoutMinMs())
               / 2;
+      Integer leaderId = configManager.getLoadManager().getRegionLeaderMap().get(regionId);
+
+      if (leaderId != -1) {
+        // The migrated node is not leader, so we specify the transfer leader to current leader node
```

Comment:
then we do not need to send rpc to datanode?

Reply:
I think so, we can't report success